plugins/Compressor/CompressorControlDialog.cpp

```diff
@@ -0,0 +1,753 @@
+	peakButton->move(m_controlsBoxX + 337, m_controlsBoxY + 248);
+
+	leftRightButton->move(m_controlsBoxX + 220, m_controlsBoxY + 231);
+	midSideButton->move(m_controlsBoxX + 220, m_controlsBoxY + 248);
+
+	compressButton->move(m_controlsBoxX + 98, m_controlsBoxY + 231);
+	limitButton->move(m_controlsBoxX + 98, m_controlsBoxY + 248);
+
+	unlinkedButton->move(m_controlsBoxX + 495, m_controlsBoxY + 180);
+	maximumButton->move(m_controlsBoxX + 495, m_controlsBoxY + 197);
+	averageButton->move(m_controlsBoxX + 495, m_controlsBoxY + 214);
+	minimumButton->move(m_controlsBoxX + 495, m_controlsBoxY + 231);
+	blendButton->move(m_controlsBoxX + 495, m_controlsBoxY + 248);
+
+	autoMakeupButton->move(m_controlsBoxX + 220, m_controlsBoxY + 206);
+	auditionButton->move(m_controlsBoxX + 658, m_controlsBoxY + 14);
+	feedbackButton->move(m_controlsBoxX + 98, m_controlsBoxY + 206);
+	m_autoAttackKnob->move(m_controlsBoxX + 460, m_controlsBoxY + 38);
+	m_autoReleaseKnob->move(m_controlsBoxX + 590, m_controlsBoxY + 38);
+	lookaheadButton->move(m_controlsBoxX + 202, m_controlsBoxY + 171);
+}
+
+// For theming purposes
+QColor const & CompressorControlDialog::inVolAreaColor() const {return m_inVolAreaColor;}
+QColor const & CompressorControlDialog::inVolColor() const {return m_inVolColor;}
+QColor const & CompressorControlDialog::outVolAreaColor() const {return m_outVolAreaColor;}
+QColor const & CompressorControlDialog::outVolColor() const {return m_outVolColor;}
+QColor const & CompressorControlDialog::gainReductionColor() const {return m_gainReductionColor;}
+QColor const & CompressorControlDialog::kneeColor() const {return m_kneeColor;}
+QColor const & CompressorControlDialog::kneeColor2() const {return m_kneeColor2;}
+QColor const & CompressorControlDialog::threshColor() const {return m_threshColor;}
+QColor const & CompressorControlDialog::textColor() const {return m_textColor;}
+QColor const & CompressorControlDialog::graphColor() const {return m_graphColor;}
+QColor const & CompressorControlDialog::resetColor() const {return m_resetColor;}
+
+void CompressorControlDialog::setInVolAreaColor(const QColor & c){m_inVolAreaColor = c;}
+void CompressorControlDialog::setInVolColor(const QColor & c){m_inVolColor = c;}
+void CompressorControlDialog::setOutVolAreaColor(const QColor & c){m_outVolAreaColor = c;}
+void CompressorControlDialog::setOutVolColor(const QColor & c){m_outVolColor = c;}
+void CompressorControlDialog::setGainReductionColor(const QColor & c){m_gainReductionColor = c;}
+void CompressorControlDialog::setKneeColor(const QColor & c){m_kneeColor = c;}
+void CompressorControlDialog::setKneeColor2(const QColor & c){m_kneeColor2 = c;}
+void CompressorControlDialog::setThreshColor(const QColor & c){m_threshColor = c;}
+void CompressorControlDialog::setTextColor(const QColor & c){m_textColor = c;}
+void CompressorControlDialog::setGraphColor(const QColor & c){m_graphColor = c;}
+void CompressorControlDialog::setResetColor(const QColor & c){m_resetColor = c;}
```

Comment:
If we use `Q_PROPERTY`'s `MEMBER` we can remove all this. These can be auto generated by moc during compile time.